CHANGES.md

```diff
@@ -1,3 +1,180 @@
+## stable-2.7.0
+
+This massive release adds support for integrating Linkerd's PKI with an external
+certificate issuer such as [`cert-manager`] as well as streamlining the
```

Comment:
You missed the link for cert-manager: https://github.com/jetstack/cert-manager

Reply:
I think this will pick up the link definition from the edge-19.10.5 section of the CHANGES.md file:

```
[`cert-manager`]: https://github.com/jetstack/cert-manager
```